pkg/build/generator/generator_test.go

```diff
@@ -1421,7 +1453,7 @@ func TestResolveImageStreamRef(t *testing.T) {
 				Name: imageRepoName + ":" + tagName,
 			},
 			expectedSuccess:   true,
-			expectedDockerRef: latestDockerReference,
+			expectedDockerRef: dockerReference,
```

Comment:
we're trying to resolve the image stream tag "testRepo:test" to the docker image reference "latestDockerReference"

the intent being to prove we probably convert imagestreamtags (which builds can't consume) to dockerimagereferences (which they can)